<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyDefineDialog</class>
 <widget class="QDialog" name="KeyDefineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabValveOutput">
    <attribute name="title">
     <string>阀输出</string>
    </attribute>
    <widget class="QGroupBox" name="grboxValveAction">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>581</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>阀动作</string>
     </property>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>358</width>
        <height>134</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QCheckBox" name="chboxValveAction0">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>卡爪</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="coboxValveAction0">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>原料1夹紧/松开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>成品1夹紧/松开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>原料2夹紧/松开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>成品2夹紧/松开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>卡爪1正转/反转</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>卡爪2正转/反转</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>反转台1正转/反转</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>反转台2夹紧/松开</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="chboxValveAction1">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>机床控制</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="coboxValveAction1">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>自动门1开/关</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>自动门2开/关</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="chboxValveAction2">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>卡盘/吹屑</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="coboxValveAction2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>卡盘1夹紧/松开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>卡盘2夹紧/松开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>吹气1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>吹气2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="grboxOutputStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>输出状态</string>
     </property>
     <widget class="QWidget" name="layoutWidget_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>321</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="chboxValveOutStatus0">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>通</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chboxValveOutStatus1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>断</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chboxValveOutStatus2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>翻转</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabReserve">
    <attribute name="title">
     <string>预留</string>
    </attribute>
    <widget class="QGroupBox" name="grboxMainBoardOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>581</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>主板输出</string>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>512</width>
        <height>102</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="verticalSpacing">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chboxMainBoardOut0">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="coboxMainBoardOut0">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Y1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y8</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="chboxMainBoardOut1">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="coboxMainBoardOut1">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Y9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y16</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="chboxMainBoardOut2">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QComboBox" name="coboxMainBoardOut2">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Y17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y21</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y22</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y23</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y24</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="chboxMainBoardOutExtend0">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="coboxMainBoardOutExtend0">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>EY1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY8</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="chboxMainBoardOutExtend1">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="coboxMainBoardOutExtend1">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>EY9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY16</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QComboBox" name="coboxMainBoardOutExtend2">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>EY17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EY20</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QCheckBox" name="chboxMainBoardOutExtend2">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="grboxReserveOutStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>输出状态</string>
     </property>
     <widget class="QWidget" name="layoutWidget_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>321</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="chboxReserveOutStatus0">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>通</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chboxReserveOutStatus1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>断</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chboxReserveOutStatus2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>翻转</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabServo">
    <attribute name="title">
     <string>伺服</string>
    </attribute>
    <widget class="QGroupBox" name="grAxisMoveDirect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>轴运动方向</string>
     </property>
     <widget class="QWidget" name="layoutWidget_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>371</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QComboBox" name="coboxAxisSelect">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>X1轴</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y1轴</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z1轴</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C轴</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A/Y2轴</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B/Z2轴</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chboxAxisDirect0">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>正向</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chboxAxisDirect1">
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>40</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>反向</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
